### ISSUE TITLE
'only_db' config option bug when db names contain underscore and are grouped

### ISSUE THREAD

**Issue by stevesmith58** [phpmyadmin/phpmyadmin#11594]
I'm using the 'only_db' config option too limit the list to just 2 databases thus :

$cfg['Servers'][$i]['only_db']= array('ink\_log','ink\_map');

This works fine in the right panel but in the the left panel it shows
all database which which begin with "ink_" (10 databases in my case)

**Comment by lem9** [comment]
Which phpMyAdmin version?

**Comment by stevesmith58** [comment]
4.5.0.2
But I think its been around since 4.0

**Comment by lem9** [comment]
I agree that there is a bug in the navigation panel (and it won't be fixed in 4.5.1). However, according to the doc, you should use
````
$cfg['Servers'][$i]['only_db']= array('ink\_log','ink\_map');
````
otherwise, in the main panel, a db named 'inkxlog' will show.

**Comment by stevesmith58** [comment]
thanks,
Yes I should have remembered about escaping.
